src/requests/index.ts

```diff
@@ -61,19 +56,6 @@ export class CoreDomainClient {
     this.export = new ExportClient(this.client);
   }
 
-  authorizeSession(sessionDetails: { email: string; sessionId: string }) {
```

Comment:
This method of authorising separately to instantiating might be the right pattern, so I'll come back to it if needed. It's possible we could call a function like this in a middleware layer to set the context for the client.